.github/workflows/daily-deploy-production.yml

```
@@ -66,21 +66,21 @@ jobs:
         id: get-latest-tag
         run: echo ::set-output name=LATEST_TAG_VERSION::$(git fetch --all --tags > /dev/null && git tag -l | sort -V --reverse | head -n 1)
 
+      - name: Get next version
```

```suggestion
      - name: Get next tag version
```
Makes it a little easier to read